release_dashboard/lib/widgets/conductor_status.dart

```diff
@@ -9,7 +9,7 @@ import 'package:provider/provider.dart';
 import 'common/tooltip.dart';
 import 'common/url_button.dart';
 
-enum SubstepEnum {
+enum StatusSubstepType {
```

Comment:
You also want to avoid putting `Type` in the name of the `Type` (unless its a generic type)